### DISCUSSION TITLE
Please Read Before Posting

### DISCUSSION THREAD

**Discussion by EricBusch** [datafeedr/wordpress-plugins#3] [Announcements]
Thanks for checking out the Datafeedr forums. 😄 

Here are some things to keep in mind when posting:

1. **Don't post sensitive data** - These forums are public and anyone on the web can see what you post. If you need to share something private, please [email us](https://datafeedr.me/contact).
1. **Choose the correct category** - We have two categories to create new threads in:
    - _Help_ - If you have a question, this is the place to ask it.
    - _Ideas_ - If you have a suggestion or feature request, this is the place to share it.
1. **Mark an answer** -  If you do post a question in the _Help_ category and someone provides you with the answer that solves your problem, don't forget to mark the reply as the "Answer".
1. **Please be kind** - Please read the [Code of Conduct](https://github.com/datafeedr/wordpress-plugins/blob/main/CODE_OF_CONDUCT.md) before posting.

If you have any questions or suggestions about these guidelines, please post below.

Thanks!

**Comment by MelBoracis** [comment]
Hello - looking for advice on choosing the right hosting SET UP.  I am not looking for a particular hosting company recommendation.  I am looking to leverage Datafeedr to build an affiliate shopping site with 3,000~5,000 products.  So with multiple product images per product that means tens of thousands of images.  However, while DF uses WooCommerce to create a full ecommerce store appearance, most of the WooCommerce functionality is not really used (no cart, no payment gateways, no inventory management, etc.).  As such, is a "WooCommerce" Hosting package that is "optimized for WooCommerce" really the appropriate hosting set up for a DF affiliate shopping site?  I know that Shared Hosting is not ideal for an affiliate store with this many products.  "WooCommerce Hosting" packages to me are not clear as to whether they are still shared hosting.  Before I start building out this store, I want to make sure I start with the right hosting set up. I don't want to spend more than necessary until the concept is proven.  Also, I'm not a programmer so non technical.  Any advice appreciated.  VPS/dedicated sounds like it's too technical.  Should I choose an intro "Cloud Hosting" package?  Or is a "WooCommerce Hosting" package appropriate even though a DF store is really just a store displaying product images with links, and not other WooCommerce functionality.  Any advice appreciated.  Lots of sites and social streams with hosting company recommendations out there (SiteGround,   WPEngine, LiquidWeb, Scala, Hostinger, Dreamhost, Kinsta, etc.), but it's the type of hosting I'm confused on what would best suit a DF affiliate store.  Thanks.

**Reply by EricBusch** [reply]
Hi

3000-5000 products is quite a few. I would recommend not importing the images. Here's how to do that: https://datafeedrapi.helpscoutdocs.com/article/237-display-external-product-image-urls-instead-of-importing-images-into-your-website

I would also consider not importing more than 3000 or else your hosting fees might get expensive.

Here's more information on that: https://datafeedrapi.helpscoutdocs.com/article/238-how-many-products-can-i-import-into-my-woocommerce-store

Hope this helps!

If you have any other questions, please don't hesitate to ask.

Eric

**Reply by MelBoracis** [reply]
OK thanks, Eric.
